src/fitnesse/wikitext/shared/ContentsItemBuilder.java

```diff
@@ -77,8 +78,27 @@ else if (hasOption("-H", Names.HELP_INSTEAD_OF_TITLE_TOC)) {
         return listItem;
     }
 
+    private int getTotalTestPagesInASuite(SourcePage page, int counter) {
```

Comment:
To me it seems easier just return 1 for test pages and sum inside the loop for a suite, then no `counter` parameter is needed.
Something like:
```
        int counter = 0;
        while (pages.hasNext()) {
           SourcePage sourcePage = pages.next();
           counter = counter + getTotalTestPagesInASuite(sourcePage);
         }
         return counter;
```

Reply:
Thanks for the comment. I changed it.